runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -200,7 +200,8 @@ L_notSync:
 L_gotHelper:
 	ldr	x0, [x27, #J9TR_SCSnippet_codeCacheReturnAddress]	// Fetch code cache EIP
 	ldr	x3, [x27, #J9TR_SCSnippet_method]
-	tbnz	x3, #1, L_USSGclinitCase			// branch if the LSB (the "clinit" bit) was set in the resolved address
+	tst	x3, #clinit_bit					// branch if the LSB (the "clinit" bit) was set in the resolved address
+	bne	L_USSGclinitCase
```

Comment:
My apologies for being obtuse, but why do you have to use two instructions to perform this bit test rather than the single test-and-branch?

Reply:
You need the bit number for the test-and-branch instruction.
i.e. the bit number is 0 when `J9TR_MethodNotCompiledBit` is 1, and it is 3 if `J9TR_MethodNotCompiledBit` is 8.
I couldn't think of a way to calculate the bit number from the value of  `J9TR_MethodNotCompiledBit` in assembler.

Reply:
Updated the file so that the checks for the clinit bit uses `tbnz` instruction.

Reply:
Ah.  I was fooled by the name.  `clinit_bit` is really the `clinit_bit_mask`, not the bit position.  The way you renamed it is fine.